fem/fespace.hpp

```diff
@@ -257,14 +278,60 @@ class FiniteElementSpace
    bool Conforming() const { return mesh->Conforming(); }
    bool Nonconforming() const { return mesh->Nonconforming(); }
 
+   /// The returned SparseMatrix is owned by the FiniteElementSpace.
    const SparseMatrix *GetConformingProlongation() const;
+
+   /// The returned SparseMatrix is owned by the FiniteElementSpace.
    const SparseMatrix *GetConformingRestriction() const;
 
+   /// The returned Operator is owned by the FiniteElementSpace.
    virtual const Operator *GetProlongationMatrix() const
    { return GetConformingProlongation(); }
+
+   /// The returned SparseMatrix is owned by the FiniteElementSpace.
    virtual const SparseMatrix *GetRestrictionMatrix() const
    { return GetConformingRestriction(); }
 
+   /// Return an Operator that converts L-vectors to E-vectors.
+   /** An L-vector is a vector of size GetVSize() which is the same size as a
+       GridFunction. An E-vector represents the element-wise discontinuous
+       version of the FE space.
+
+       The layout of the E-vector is: ND x VDIM x NE, where ND is the number of
+       degrees of freedom, VDIM is the vector dimension of the FE space, and NE
+       is the number of the mesh elements.
+
+       The parameter @a e_ordering describes how the local DOFs in each element
+       should be ordered, see ElementDofOrdering.
+
+       For discontinuous spaces, where the element-restriction is the identity,
+       this method will return NULL.
```

Comment:
Why not return the Identity Operator? We are carrying a `NULL` pointer that is going to be handled in each part where restriction is used: here and Laghos for example.